Description: Debian packaging uses gtk-update-icon-cache-3.0, update path in configure.ac.
Author: Martin Pitt <mpitt@debian.org>

---
 configure.ac |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/configure.ac
+++ b/configure.ac
@@ -71,9 +71,9 @@ AC_SUBST(GTK2_VERSION)
 GLIB_COMPILE_RESOURCES=`$PKG_CONFIG --variable glib_compile_resources gio-2.0`
 AC_SUBST(GLIB_COMPILE_RESOURCES)
 
-AC_PATH_PROG(GTK_UPDATE_ICON_CACHE, gtk-update-icon-cache)
+AC_PATH_PROG(GTK_UPDATE_ICON_CACHE, gtk-update-icon-cache-3.0)
 if test -z "$GTK_UPDATE_ICON_CACHE"; then
-    AC_MSG_ERROR([Could not find gtk-update-icon-cache])
+    AC_MSG_ERROR([Could not find gtk-update-icon-cache-3.0])
 fi
 
 # Workaround to make aclocal get the right flags
